### ISSUE TITLE
Nothing provides libstdc++.so.6(CXXABI_1.3.15)(64bit) needed by gcc-latest-14.0.1-3.20240114git5f6557b3c611.fc39.x86_64

### ISSUE THREAD

**Issue by Mno-hime** [jwakely/pkg-gcc-latest#17]
Updating the `gcc-latest` package on Fedora 39 from the `copr:copr.fedorainfracloud.org:jwakely:gcc-latest` repo fails with:

```
nothing provides libstdc++.so.6(CXXABI_1.3.15)(64bit) needed by gcc-latest-14.0.1-3.20240114git5f6557b3c611.fc39.x86_64 from copr:copr.fedorainfracloud.org:jwakely:gcc-latest
```

I am updating on top of `gcc-latest-14.0.0-4.20231112git93e92b2e5d68.fc39.x86_64`.

**Comment by jwakely** [comment]
This is fixed now. I changed which machine I was using to start the builds, and I hadn't updated the repo to pull the #16 fix :facepalm:

**Comment by jwakely** [comment]
The package version of the last few builds moved backwards from 14.0.0-4 to 14.0.0-3, so `dnf` didn't try to update to the broken packages. That's why we didn't notice sooner. When the upstream GCC version changed from 14.0.0 to 14.0.1 we got a 14.0.1-3 build which is newer than 14.0.0-4 so `dnf` tried to use it.

This shouldn't happen again, temporarily building on a different machine was due to moving home, so a one-off.
